src/main/java/io/zold/api/Wallets.java

```diff
@@ -32,7 +34,9 @@
 public interface Wallets extends Iterable<Wallet> {
     /**
      * Create a wallet.
-     * @return The new wallet
+     * @param id The new wallet id.
+     * @return The new wallet.
+     * @throws IOException If an error occurs.
      */
-    Wallet create();
+    Wallet create(final long id) throws IOException;
```

Comment:
@victornoel see my thoughts [here](https://github.com/zold-io/java-api/issues/32#issuecomment-404681557). I think it's best for the id to be generated internally. I know that in zold-io/zold they have that separate `Id` class but I just don't see the need for that here at the moment.

Reply:
@llorllale good for me